### PR TITLE
Update 2012 packets

### PR DESCRIPTION
**Pull request by OliverSpeed**
The rest have different data compared to other revisions.

### CODE REVIEW

File: src/Skylight.Protocol.X/Skylight.Protocol.RELEASE63-201211141113-913728051/packets.json [hunk 1]
```diff
@@ -2,6 +2,10 @@
   "Revision": "RELEASE63-201211141113-913728051",
   "Protocol": "Modern",
   "Incoming": {
+    "Advertisement.GetInterstitial": {
+      "Id": 2346,
```
Comment (aromaa) [inline src/Skylight.Protocol.X/Skylight.Protocol.RELEASE63-201211141113-913728051/packets.json:10]:
```suggestion
      "Id": 3080,
```
2346 is GetRoomAd

File: src/Skylight.Protocol.X/Skylight.Protocol.WIN63-202111081545-75921380/packets.json [hunk 2]
```diff
@@ -1552,13 +1552,13 @@
...
-          "Name": "Links"
+          "Type": "int",
+          "Name": "TrackingId"
         },
         {
           "Mapping": "Field",
-          "Type": "int",
-          "Name": "TrackingId"
+          "Type": "List<Link>",
+          "Name": "Links"
```
Comment (aromaa) [inline src/Skylight.Protocol.X/Skylight.Protocol.WIN63-202111081545-75921380/packets.json]: Mistake? This should be reverted, this is wrong

**Review by aromaa — CHANGES_REQUESTED** [review]

File: src/Skylight.Protocol.X/Skylight.Protocol.WIN63-202111081545-75921380/packets.json (hunk 2 above)
Comment (OliverSpeed) [reply src/Skylight.Protocol.X/Skylight.Protocol.WIN63-202111081545-75921380/packets.json]: indeed a mistake

File: src/Skylight.Protocol.X/Skylight.Protocol.RELEASE63-201211141113-913728051/packets.json (hunk 1 above)
Comment (OliverSpeed) [reply src/Skylight.Protocol.X/Skylight.Protocol.RELEASE63-201211141113-913728051/packets.json:10]: done

**Review by aromaa — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**